src/common/Button/button.scss

```diff
@@ -0,0 +1,150 @@
+@import '../../scss/variables';
+@import '../../scss/colors';
+@import '../../scss/borders';
+
+.btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 90px;
+  height: 36px;
+  padding: 0 16px;
+  color: $white;
+  font-weight: 500;
+  font-size: 14px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  line-height: 16px;
+  border: 0;
```

Comment:
If you put a longer label on the button, you can see that when it becomes focused, it moves a little bit:
![button-border](https://user-images.githubusercontent.com/13918850/110664157-d8c5ff00-81cf-11eb-826a-0d6bebd461fc.gif)

This happens because when not in focus it has no border at all, then when it is focused it does have a border with 1px width (on each side).
The solution here is to add a transparent border when the button is not focused.
Then when focused you can only change the border color so it becomes visible.

Here you need:
```suggestion
  border: 1px solid transparent;
```
Later, when you have things like:
```js
border: 1px solid rgba($black, 0.4);
```
You can leave them as-is, but you can also change to:
```js
border-color: rgba($black, 0.4);
```
because the width and style of the border remains unchanged (`1px` and `solid`).
I leave it up to you to decide.